demo/apisix/conf/apisix.yaml

```diff
@@ -91,13 +91,14 @@ services:
             - - request_method
               - "~="
               - OPTIONS
-        client_id: "james_apisix"
-        client_secret: "james_apisix"
+        client_id: "james"
+        client_secret: "james"
         discovery: "http://sso.example.com/.well-known/openid-configuration"
         scope: "openid profile"
         bearer_only: true
         introspection_endpoint_auth_method: "client_secret_post"
         redirect_uri: "http://test.sso.example.com:8080/login-callback.html"
+        use_jwks: true
```

Comment:
If we choice jwks, and introspection is redundant ( the fact is it does not work with public =1), We should remove introspection relate